test/controllers/users_controller_test.js

```diff
@@ -3,22 +3,30 @@ const usersModel = require("../../src/models/users_model");
 const chai = require("chai"),
     chaiHttp = require("chai-http");
 const should = chai.should();
-const {expect} = chai;
 
 chai.use(chaiHttp);
 
 describe("Endpoints exists for users", () => {
+    before((done) => {
```

Comment:
adding this function here so that users are created in the begining instead of in each test case. I will work on this more in a different PR so that all users are created in the begining.